### DISCUSSION TITLE
请问字数统计的口径是什么？

### DISCUSSION THREAD

**Discussion by chenhongyang113** [sjtug/SJTUThesis#805] [提问] [answered]
请教一下：
.\Compile.bat wordcount 
统计的字数，有[纯中文]和[中文+英文]两个数据，但是将编译好的PDF文件用福昕阅读器转成Word后，纯中文的字数有比较明显的差距，目前的情况是：
LaTeX统计的纯中文是32000字，但是福昕转的Word是37000字，差距比想象的大，那LaTeX命令里是没把哪部分字统计进来吗？
谢谢！

**Comment by LogCreative** [comment, marked as answer]
使用 [TeXcount](http://mirrors.sjtug.sjtu.edu.cn/CTAN/support/texcount/doc/TeXcount.pdf) 计算字数。[命令行参数](https://app.uio.no/ifi/texcount/documentation.html) 显示纯中文字数将不包含英文词。

https://github.com/sjtug/SJTUThesis/blob/b164d0f5b3087ed86c1038b5a85014b47a92c6f2/Compile.bat#L90

其中会被 TeXcount 忽略的内容以下行开始
https://github.com/sjtug/SJTUThesis/blob/b164d0f5b3087ed86c1038b5a85014b47a92c6f2/main.tex#L17
并直到下行结束
https://github.com/sjtug/SJTUThesis/blob/b164d0f5b3087ed86c1038b5a85014b47a92c6f2/main.tex#L45

默认配置下，只计算正文部分除参考文献外的字数。

**Reply by chenhongyang113** [reply]
谢谢指导！
